pootle/core/initdb.py

```diff
@@ -268,7 +274,7 @@ def create_default_projects():
                  'guide</a>.</div>'),
         'virtual_path': "announcements/projects/"+tutorial.code,
     }
-    ann = Announcement(**criteria)
+    ann = StaticPage(**criteria)
```

Comment:
Hmm just saw the change in the import. Still, I don't believe it's necessary the rename, and it's completely irrelevant in the context of this PR.